doc/source/roadmap-detailed.rst

```diff
@@ -168,6 +161,27 @@ Ideas for new features:
 - Add type-generic wrappers in the Cython BLAS and LAPACK
 - Make many of the linear algebra routines into gufuncs
 
+**BLAS and LAPACK**
+
+The Python and Cython interfaces to BLAS and LAPACK in ``scipy.linalg`` are one
+of the most important things that SciPy provides. In general ``scipy.linalg``
+is in good shape, however we can make a number of improvements:
+
+1. Library support. Our released wheels now ship with OpenBLAS, which is
+currently the only feasible performant option (ATLAS is too slow, MKL cannot be
```

Comment:
For this to render properly it needs to be indented. I checked the build locally for this, might be worth just doing `[skip travis] [skip appveyor]` in the commit message so that CircleCI builds and has a chance to complain about any issues.

Reply:
![Screenshot from 2020-05-04 10-53-55](https://user-images.githubusercontent.com/2365790/80979607-92588480-8df5-11ea-99d6-ee353378210f.png)

Reply:
Oh I saw that but it didn't bother me the way it looks now. Happy to indent it though.

Reply:
fixed